gameoflife.php

```diff
@@ -97,15 +97,15 @@ function autoload ($className)
 $maxSteps = 100;
 if ($options->getOption("maxSteps"))
 {
-    echo $maxSteps;
+    $maxSteps = $options->getOption("maxSteps");
 }
 
 /**
  * Possible boards
  */
-//$life->generateRandomBoard();
+$life->generateRandomBoard();
 //$life->generateGleiter();
-$life->generateBlinker();
+//$life->generateBlinker();
```

Comment:
Why is this code even in there? You now have command-line parameters to define with which board this should start. And if you have a blinker, you should also add an option startBlinker with which you can start a blinker!